tests/manage/pv_services/pvc_resize/test_pvc_expansion.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, tier1
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.5')
+@pytest.mark.polarion_id('OCS-302')
+class TestPvcExpand(ManageTest):
```

Comment:
can we consolidate the methods in this class with tests.manage.pv_services.test_pvc_resize.TestPvcResize?

Reply:
The plan is to add more tier1 test cases cases under this test class. I will try to add larger test cases in separate PRs. I will also move test_pvc_resize to this module in a separate PR.

Reply:
+1 thanks!